src/components/SQForm/SQFormDateTimePicker.tsx

```diff
@@ -1,19 +1,37 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import moment from '@material-ui/pickers/adapter/moment';
```

Comment:
Is this the same thing as `import moment from 'moment'`?

Reply:
It is not the same, as the adapter is not the same as the moment library itself. MUI supplies the adapter I believe for typing. I want to find some docs to be more sure.

Reply:
It looks like I am off base. I am not sure where I pulled my original line from. I will update it.